addons/europeans_tribe.wad/productionsites/sheepfarm/init.lua

```diff
@@ -58,7 +58,7 @@ descriptions:new_productionsite_type {
          }
       },
       produce_wool_basic = {
-         -- TRANSLATORS: Completed/Skipped/Did not start breeding sheep because ...
+         -- TRANSLATORS: Completed/Skipped/Did not start producing wool because ...
          descname = _"produce wool",
```

Comment:
```suggestion
         descname = _"producing wool",
```
Same below, and in several other files there are similar glitches. Please always use `producing` instead of `produce` in those program names.

Reply:
Solved.